tests/helpers/volumes.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) IBM Corporation 2023
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import pytest
+
+class Volume:
+# Class volume to save the information with volume name and a flag of use
+# to mange in all test to not harcoded the values of the array.
+    def __init__(self, name):
+        self.name = name
+        self.in_use = False
+
+    def __str__(self):
+        return f'The volume {self.name} is in {self.in_use} in use'
+
+    def use(self):
+        self.in_use = True
+
+    def free(self):
+        self.in_use = False
+
+class ls_Volume:
+# Class to call fixture get_volumes and filled the array and manage volumes
+# instances of are in use or not
+    def __init__(self, *get_volumes):
+        self.volume = get_volumes
```

Comment:
self.volume is a list, is better to change the variable name to volume_list but in this case I will recommend converting the output of get_volumes to a set or dict, that way we can avoid having to do a for loop every time we want to access an specific volume.

Reply:
The code has since been changed, best guess is this is addressed when the code now has `all_volumes = hosts.all.zos_operator(cmd="d u,dasd,online,,65536")`